

# Fighting against evolution of antibiotic resistance by utilizing evolvable antimicrobial drugs

Mehmet Fatih Cansizoglu<sup>1</sup> · Erdal Toprak<sup>1,2</sup>

Received: 24 April 2017 / Revised: 3 May 2017 / Accepted: 5 May 2017 / Published online: 11 May 2017  
© Springer-Verlag Berlin Heidelberg 2017

**Abstract** Antibiotic resistance is a worldwide public health problem (Bush et al. in Nat Rev Microbiol 9:894–896, 2011). The lack of effective therapies against resistant bacteria globally leads to prolonged treatments, increased mortality, and inflating health care costs (Oz et al. in Mol Biol Evol 31:2387–2401, 2014; Martinez in Science 321:365–367, 2008; Lipsitch et al. in Proc Natl Acad Sci USA 97:1938–1943, 2000; Taubes in Science 321:356–361, 2008; Laxminarayan et al. in Lancet, 2016; Laxminarayan et al. in Lancet Infect Dis 13:1057–1098, 2013). Current efforts towards a solution of this problem can be boiled down to two main strategies: (1) developing of new antimicrobial agents and (2) searching for smart strategies that can restore or preserve the efficacy of existing antimicrobial agents. In this short review article, we discuss the need for evolvable antimicrobial agents, focusing on a new antimicrobial technology that utilizes peptide-conjugated phosphorodiamidate morpholino oligomers to inhibit the growth of pathogenic bacteria by targeting bacterial genes.

**Keywords** Antibiotic resistance · Evolution · Microbial evolution · Gene silencing · Peptide-conjugated phosphorodiamidate morpholino oligomers (PPMOs)

## Introduction

Antibiotics are small molecules that inhibit bacterial growth by blocking key cellular pathways of bacteria in order to treat and prevent bacterial infections (Fig. 1). Historically, antibiotic compounds have been either isolated from microorganisms such as fungi or engineered to block catalytic activity of target enzymes. Development of these compounds is often a long and expensive process and due to natural and economical limitations, the number of approved novel antimicrobial compounds has been declining in the past decade. This problem is further exacerbated by the fact that humankind has not been successful at preserving the efficacy of existing antibiotic compounds due to irrational use of antibiotics, which promoted evolution of resistance. Resistance against every single antibiotic drug, regardless of the drugs' mechanisms of action, was reported very soon after the introduction of these drugs to clinical use (Fig. 1).

A rational method of antibiotic use that clears infections without promoting antibiotic resistance and disturbing healthy host microbiota would be a paradigm shift in medicine. Recently, there has been a growing interest into designing strategies for using existing drugs more efficiently, such as using antibiotics in combinations or cycling antibiotics in hospital settings (Baym et al. 2016). Most of these strategies rely on observations in laboratory evolution experiments or theoretical predictions (Bergstrom et al. 2004; Hegreness et al. 2008; Gonzales et al. 2015). Alas, none of the proposed methodologies have yet been successful at impeding evolution of resistance. Furthermore, both single antibiotic compounds and antibiotic combinations can unselectively kill both pathogenic and commensal bacteria in the host and significantly perturb healthy human

---

Communicated by M. Kupiec.

---

✉ Erdal Toprak  
Erdal.Toprak@UTSouthwestern.edu

<sup>1</sup> Green Center for Systems Biology, University of Texas Southwestern Medical Center, Dallas, TX, USA

<sup>2</sup> Department of Pharmacology, University of Texas Southwestern Medical Center, Dallas, TX, USA

|                                 | <img alt="DNA double helix icon" data-bbox="541 164 695 263" src="78c10c6c1104aea28089918a14d33a59_img.jpg"/> <b>Antibiotic Sensitive</b>                                                                     | <img alt="DNA double helix icon with mutation" data-bbox="959 164 1113 263" src="a3cddb30971612cb7efe382e74f4b8e3_img.jpg"/> <b>Antibiotic Resistant</b>                                     |
|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <b>ANTIBIOTICS</b>              | Inhibit bacterial growth by blocking key cellular pathways<br><img alt="Pill icon" data-bbox="700 297 843 438" src="5d227ca103fb08648c0064ca1be1a56c_img.jpg"/> <b>✓ Effective</b>                            | New compounds with novel targeting mechanisms needed<br><img alt="Pill icon" data-bbox="1133 297 1276 438" src="052d0bcc0326e6e37e87b2aa6beca4c9_img.jpg"/> <b>✗ Ineffective</b>             |
| <b>GENE EDITING</b>             | <img alt="Gene editing icon" data-bbox="470 487 641 634" src="75bfc303bf31df2700a87766c4d636df_img.jpg"/> Essential or resistance genes are targeted<br><b>✓ Effective</b>                                    | <img alt="Gene editing icon" data-bbox="978 487 1149 634" src="5705323302a95d1eeacc802504ff2e9f_img.jpg"/> Can be redesigned to target resistance conferring mutations<br><b>✓ Effective</b> |
| <b>GENE SILENCING WITH PPMO</b> | Designed to block translation of essential or resistance genes in pathogenic bacteria<br><img alt="PPMO icon" data-bbox="700 750 901 902" src="41c5786182c2a5cb71bb079068b87749_img.jpg"/> <b>✓ Effective</b> | Can be redesigned to target resistance conferring mutations<br><img alt="PPMO icon" data-bbox="1133 750 1335 902" src="0afebdce32a80a5822189dba1221fd1c_img.jpg"/> <b>✓ Effective</b>        |

**Fig. 1** Classical and novel antibacterial strategies for treating antibiotic-sensitive and -resistant pathogens. Antibiotics are not effective once bacteria acquire resistance and therefore these drugs need to be either modified or replaced by novel antibiotics for demanded effi-

cacy. Gene editing and gene silencing based methods are also prone to losing their efficacy due to evolved resistance. However, both tools can be quickly redesigned to effectively work against resistant bacteria

microbiome leading to dysbiosis with severe complications. Therefore, there is a great need for new antimicrobial treatment platforms that can rapidly deliver evolvable and custom tailored drugs that eradicate pathogens in a sequence-specific way.

Classical antibiotic treatments aim to inhibit bacterial growth at the protein level by blocking catalytic activities of target enzymes that have critical roles in bacterial physiology. For instance, quinolones inhibit bacterial growth by blocking DNA gyrase activity, which is crucial for DNA replication. These drugs can inhibit the growth of antibiotic-sensitive pathogens but they also nonspecifically inhibit commensal bacteria throughout the treatment. Moreover, resistance to quinolones emerges rapidly and single amino acid replacements in the DNA gyrases, such as S83L or D87N replacements in the GyrA enzyme of *Escherichia coli*, are enough to render some of quinolones completely ineffective (Oz et al. 2014). If resistance emerges as a result of mutations in the target enzyme, an obvious solution would be to modify the antibiotic compounds such that they can still inhibit mutated enzyme activities. However, this process is fairly slow and still costly since it requires structural analysis of the mutated enzymes and accordingly developing and testing several

drug variants. Alternatively, inhibition of GyrA activity in the targeted pathogen could have been done at the DNA or RNA levels by disrupting or silencing the *gyrA* gene, respectively. This way, it would have been possible to engineer sequence-specific narrow spectrum agents for clearing infections. This goal can now be achieved with the help of novel gene targeting technologies.

The last decade has been transformative in biology. With the remarkable advances in sequencing, gene manipulation methods and high-throughput technologies, a significant amount of knowledge about genes, gene functions, and their regulation has been made available (Nichols et al. 2011; Laabei and Massey 2016; Brinsmade 2016). Several comprehensive gene deletion or transposon libraries for several bacterial species have been generated and in many research labs across the world, it is a standard practice to manipulate bacterial genomes and silence bacterial genes of interest (Nichols et al. 2011; Baba et al. 2006; Datsenko and Wanner 2000). In principle, any bacterial gene can be either knocked down or knocked out in laboratory settings. As a result, it becomes possible to kill bacteria by targeting their essential genes (i.e., polymerase or ribosomal genes) or to restore antibiotic sensitivity of resistant bacteria by targeting resistance related genes (i.e., efflux

or beta lactamase genes) with the help of gene targeting/editing technologies (Bikard et al. 2013; Sontheimer and Marraffini 2010; Orndorff 2016). The biggest advantage of such methods is their evolvability. Advancements in gene sequencing technologies now allow us to quickly identify resistance-conferring genetic changes and modify the gene targeting tools in a sequence-specific way such that the mutated genes can still be targeted (Fig. 1). For instance, it is possible to utilize gene silencing tools to specifically target mutated genes and even steer evolution towards antibiotic sensitivity by selecting the wild type genes. Gene editing technologies are very promising but currently far from being used in the clinics due to several shortcomings. First, delivering the reagents necessary for gene targeting into the human host imposes significant health risks for the already sick patients. Second, gene editing methods typically rely on the use of selection markers that are used for a particular phenotype such as antibiotic resistance and it is not always obvious to utilize gene targeting methods to deselect for resistant bacteria. Finally, there is a legitimate concern about using some of the gene delivery vehicles, particularly viruses, as they can potentially evolve and become human pathogens.

Towards modulating antibiotic resistance in pathogenic bacteria, we recently utilized peptide-conjugated phosphorodiamidate morpholino oligomers (PPMOs) as sequence-specific antisense molecules for enhancing efficacy of existing antimicrobial drugs (Greenberg et al. 2010; Geller et al. 2005, 2013; Meng et al. 2015; Tilley et al. 2006, 2007). Briefly, PPMOs are synthetic oligos made from 6-sided morpholine rings joined together by phosphorodiamidate linkages. Each morpholine ring has a nucleotide attached to it. PPMOs can be designed to bind to complementary sequences in mRNA molecules and block protein translation (Fig. 1). Unlike short RNA molecules that are also considered as therapeutic agents, PPMOs can stay intact in the cell since RNase enzymes cannot hydrolyze PPMOs. PPMOs can be designed to have insignificant toxicity against mammalian cells and can be delivered to both eukaryotic cells and prokaryotic cells with the help of the peptides conjugated to them (Greenberg et al. 2010; Geller et al. 2013). Previously, PPMOs were successfully used to target essential genes in several bacterial species. They were also shown to inhibit translation of resistance genes (Geller et al. 2013; Howard et al. 2017). In our recently published article, we proposed to use PPMOs to enhance the efficacy of antibiotics by silencing resistance genes that cause intrinsic or acquired antibiotic resistance in pathogenic bacteria. Specifically, we designed a PPMO that silences the *acrA* gene in *E. coli*, *Klebsiella pneumoniae* and *Salmonella enterica* (Ayhan et al. 2016).

The AcrA protein partners with AcrB and TolC proteins to form the AcrAB-TolC efflux pump complex that excretes

antibiotic molecules from the cytoplasm of several Gram negative bacteria (Blair et al. 2014). The presence of the AcrAB-TolC complex makes Gram negative bacteria intrinsically resistant to numerous antibiotic compounds such as macrolides and some beta lactams. Also, overexpression of the AcrA/AcrB proteins and point mutations in AcrB are commonly found in multidrug resistant bacteria (Oz et al. 2014; Toprak et al. 2012; Blair et al. 2015). Therefore, we hypothesized that the silencing of the *acrA*, *acrB* or *tolC* genes would decrease antibiotic resistance of bacteria and enhance efficacy of several antibiotic compounds. Indeed, we have found that particularly silencing the *acrA* gene reduced the minimum inhibitory concentrations (MIC) of more than 10 antibiotics up to 50-fold (Ayhan et al. 2016). This way, we were able to kill *E. coli*, *Klebsiella pneumoniae* and *Salmonella enterica* cells using low antibiotic doses that are normally sub inhibitory. Furthermore, we were able to amplify efficacy of antibiotic combinations by the help of PPMOs. Strikingly, this approach allowed the combined use of even antagonistic antibiotic pairs that are known to dampen each other's activity (Blair et al. 2014). Although clinically not favored, it was previously shown that the use of antagonistic antibiotic pairs could impede evolution of resistance (Hegreness et al. 2008).

The use of PPMOs and similar technologies in the clinic as antimicrobials or antibiotic enhancers will be a significant advancement in medicine. Since the PPMO sequences can be modified to silence even genes with resistance-conferring mutations, it will be possible to efficiently target antibiotic susceptible or resistant bacteria in a sequence-specific way. The PPMO sequences can be engineered for minimizing the nonspecific targeting of host cells and commensal bacteria in the host. In our experience, the *acrA*-PPMO had no detectable toxicity against the human cell line we tested (Ayhan et al. 2016). Encouragingly, PPMOs have recently been FDA approved for clinical trials for the treatment of *Duchenne's Muscular Dystrophy* (DMD). However, like any new biological tools, there are several limitations of the current PPMO technologies. In our experience, relatively high doses of PPMOs (5–10  $\mu\text{M}$ ) are typically necessary for silencing the targeted gene. Given that PPMOs are large molecules (10–15 base pairs), it might be challenging to create such high PPMO concentrations in the human body without causing toxicities. Finally, resistance to PPMOs can also evolve. It was recently reported that a mutation in an active transport gene in *E. coli* reduced bacterial membrane permeability for PPMOs and hence increased resistance to PPMOs. Fortunately, the resistance was reverted when the peptide attached to the PPMO molecules was replaced with another peptide.

We will be hearing more about the use of gene editing methods including the use of PPMOs in the near future. Once the problems we briefly mentioned are addressed,

for the first time in human history, we will have evolvable drugs that can compete with the evolution of resistance in pathogens. The use of sequencing-based population genetic models together with evolvable gene editing and silencing tools will give us the opportunity to outsmart pathogens and effectively treat infectious diseases.

**Acknowledgements** Authors would like to thank Dr. Miriam Osterfield for her careful and critical reading of the manuscript.

## References

Ayhan DH et al (2016) Sequence-specific targeting of bacterial resistance genes increases antibiotic efficacy. *PLoS Biol* 14(9):e1002552

Baba T et al (2006) Construction of *Escherichia coli* K-12 in-frame, single-gene knockout mutants: the Keio collection. *Mol Syst Biol* 2:2006.0008

Baym M, Stone LK, Kishony R (2016) Multidrug evolutionary strategies to reverse antibiotic resistance. *Science* 351(6268):3292

Bergstrom CT, Lo M, Lipsitch M (2004) Ecological theory suggests that antimicrobial cycling will not reduce antimicrobial resistance in hospitals. *Proc Natl Acad Sci USA* 101(36):13285–13290

Bikard D et al (2013) Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system. *Nucleic Acids Res* 41(15):7429–7437

Blair JM et al (2015) AcrB drug-binding pocket substitution confers clinically relevant resistance and altered substrate specificity. *Proc Natl Acad Sci USA* 112(11):3511–3516

Blair JM, Richmond GE, Piddock LJ (2014) Multidrug efflux pumps in Gram-negative bacteria and their role in antibiotic resistance. *Future Microbiol* 9(10):1165–1177

Brinsmade SR (2016) CodY, a master integrator of metabolism and virulence in Gram-positive bacteria. *Curr Genet* 63(3):417–425. doi:10.1007/s00294-016-0656-5

Bush K et al (2011) Tackling antibiotic resistance. *Nat Rev Microbiol* 9(12):894–896

Datsenko KA, Wanner BL (2000) One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. *Proc Natl Acad Sci USA* 97(12):6640–6645

Geller BL et al (2005) Antisense phosphorodiamidate morpholino oligomer inhibits viability of *Escherichia coli* in pure culture and in mouse peritonitis. *J Antimicrob Chemother* 55(6):983–988

Geller BL et al (2013) Gene-silencing antisense oligomers inhibit acinetobacter growth in vitro and in vivo. *J Infect Dis* 208(10):1553–1560

Gonzales PR et al (2015) Synergistic, collaterally sensitive beta-lactam combinations suppress resistance in MRSA. *Nat Chem Biol* 11(11):855–861

Greenberg DE et al (2010) Antisense phosphorodiamidate morpholino oligomers targeted to an essential gene inhibit Burkholderia cepacia complex. *J Infect Dis* 201(12):1822–1830

Hegreness M et al (2008) Accelerated evolution of resistance in multidrug environments. *Proc Natl Acad Sci USA* 105(37):13977–13981

Howard JJ et al (2017) Inhibition of pseudomonas aeruginosa by peptide-conjugated phosphorodiamidate morpholino oligomers. *Antimicrob Agents Chemother* 61(4):e01938-16

Laabei M, Massey R (2016) Using functional genomics to decipher the complexity of microbial pathogenicity. *Curr Genet* 62(3):523–525

Laxminarayan R et al (2013) Antibiotic resistance—the need for global solutions. *Lancet Infect Dis* 13(12):1057–1098

Laxminarayan R et al (2016) Access to effective antimicrobials: a worldwide challenge. *Lancet* 387(10014):168–175

Lipsitch M, Bergstrom CT, Levin BR (2000) The epidemiology of antibiotic resistance in hospitals: paradoxes and prescriptions. *Proc Natl Acad Sci USA* 97(4):1938–1943

Martinez JL (2008) Antibiotics and antibiotic resistance genes in natural environments. *Science* 321(5887):365–367

Meng J et al (2015) Reversion of antibiotic resistance by inhibiting mecA in clinical methicillin-resistant *Staphylococci* by antisense phosphorothioate oligonucleotide. *J Antibiot (Tokyo)* 68(3):158–164

Nichols RJ et al (2011) Phenotypic landscape of a bacterial cell. *Cell* 144(1):143–156

Orndorff PE (2016) Use of bacteriophage to target bacterial surface structures required for virulence: a systematic search for antibiotic alternatives. *Curr Genet* 62(4):753–757

Oz T et al (2014) Strength of selection pressure is an important parameter contributing to the complexity of antibiotic resistance evolution. *Mol Biol Evol* 31(9):2387–2401

Sontheimer EJ, Marraffini LA (2010) Microbiology: slicer for DNA. *Nature* 468(7320):45–46

Taubes G (2008) The bacteria fight back. *Science* 321(5887):356–361

Tilley LD et al (2006) Gene-specific effects of antisense phosphorodiamidate morpholino oligomer-peptide conjugates on *Escherichia coli* and *Salmonella enterica* serovar typhimurium in pure culture and in tissue culture. *Antimicrob Agents Chemother* 50(8):2789–2796

Tilley LD et al (2007) Antisense peptide-phosphorodiamidate morpholino oligomer conjugate: dose-response in mice infected with *Escherichia coli*. *J Antimicrob Chemother* 59(1):66–73

Toprak E et al (2012) Evolutionary paths to antibiotic resistance under dynamically sustained drug selection. *Nat Genet* 44(1):101–105